src/overmap.cpp

```diff
@@ -2370,9 +2370,10 @@ void overmap::move_hordes()
             if(add_to_group == NULL) {
                 mongroup m(GROUP_ZOMBIE, p.x, p.y, p.z, 0, 1);
```

Comment:
0 currently signifies that the group can be deleted, but I could change that check to also check the list for emptiness.